Cargo.toml

```diff
@@ -19,7 +19,10 @@ bitflags = "1.0.1"
 serde = { version = "1.0.34", features = ["derive"] }
 serde_json = "1.0.50"
 serde_repr = "0.1"
-url = {version = "2.0.0", features = ["serde"]}
+# https://github.com/yescallop/fluent-uri-rs/pull/9
+# https://github.com/yescallop/fluent-uri-rs/pull/10
+fluent-uri = { git = "https://github.com/gibbz00/fluent-uri-rs", branch = "mixed_features"}
```

Comment:
If these PRs are needed they need to get publish to crates.io before this can be merged. The LSP spec do say that they use RFC 3986 URIs so the change seems warranted though https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#uri

Reply:
Since you already added a newtype wrapper you could implement the added traits on that newtype to avoid the need for those PRs.

Reply:
🤙